canonical-web-and-design/maas-ui
# Add a shared footer

Done:
- Add a footer component.
- Display the footer in both the angular and react clients. Fixes: https://github.com/canonical-web-and-design/maas-ui/issues/280.
- Drive-by fix to add lint/test to the top level package.

QA:
- Login and view the react and angular clients.
- Both should display the footer.
